### PR TITLE
fix lint warnings from DrawerLayoutAndroid

### PR DESCRIPTION
**Pull request by trave7er**
fix 3 lint warnings from DrawerLayoutAndroid.android.js

### CODE REVIEW

File: Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js [hunk 1]
```diff
@@ -21,8 +20,6 @@ var View = require('View');
 var DrawerConsts = UIManager.AndroidDrawerLayout.Constants;
 
 var dismissKeyboard = require('dismissKeyboard');
-var merge = require('merge');
-var requireNativeComponent = require('requireNativeComponent');
```
Comment (kmagiera) [inline Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js]: This is being referenced in line no 222

**Comment by kmagiera** [comment]
Can you please make sure that UIExplorer works with your changes applied, it looks like you're removing require for the module that is still used, which should result in a redbox on an attempt to run UIExplorer on android. Thanks!

**Comment by trave7er** [comment]
sorry for the mistake.

**Comment by mkonicek** [comment]
@facebook-github-bot shipit

**Comment by skevy** [comment]
@mkonicek this one sunk like the titanic...maybe an iceberg appeared that day? I'll try again.

**Comment by skevy** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**